spectroscopy/code_src/spitzer_functions.py

```diff
@@ -47,7 +47,7 @@ def SpitzerIRS_get_spec(sample_table, search_radius_arcsec, COMBINESPEC):
 
         # If multiple entries are found, pick the closest.
         # Or should we take the average instead??
-        if len(tab) > 0:
+        if len(tab) > 1:
             print("More than 1 entry found", end="")
             if not COMBINESPEC:
                 print(" - pick the closest")
```

Comment:
This code change:
- appears to be what the original author intended. (```print("More than 1 entry found"```)
- does *not* change the results but does make the code more efficient. (The code block that finds the closest entry now gets skipped when `len(tab) == 1`.)

Reply:
It makes sense to me that this is a typo and not intentional.  I agree with the change.

Along the same lines, do we need the 'else' statement for this 'if' statment?  since we have already covered the case where it is 0, it should never get to that else.  Which may mean that we can get rid of the whole 'if len(tab)> 1 ...else' and reduce the indentation???  Somehow I can't leave this comment on the appropriate line, sorry.

Reply:
> Along the same lines, do we need the 'else' statement for this 'if' statement?

That code block wasn't getting hit originally but it will now when `len(tab) == 1`. And/but I agree that this could be made simpler (== easier to follow).